index.html

```diff
@@ -1,169 +1,343 @@
+      >
+        <ul>
+          <li>
+            <label for="name" class="form-label">Your name</label>
+            <input
+              class="form-input"
+              type="text"
+              name="full-name"
+              id="name"
+              maxlength="30"
+              placeholder="name"
+              required
+            />
+          </li>
+          <li>
+            <label for="email" class="form-label">Your email</label>
+            <input
+              class="form-input"
+              type="email"
+              id="email"
+              name="email"
+              placeholder="email"
+              required
+            />
+          </li>
+          <li>
+            <textarea
+              class="form-message-input"
+              name="message"
+              id="message"
+              maxlength="500"
+              placeholder="Write your message here"
+            ></textarea>
+          </li>
+          <li>
+            <input
+              class="form-submut-btn"
+              type="submit"
+              name="submit"
+              value="Start collaboration"
+            />
+          </li>
+        </ul>
+      </form>
+      <footer class="wrapper-footer">
+        <div class="footer-title">
+          <h3 class="title-footer">Get my Resume</h3>
+          <a href="#"
+            ><img
+              class="icon-link-teal"
+              src="/03_Resources/01_Graphics/02_Stock/ic_link_teal.svg"
+              alt="Link teal icon"
+          /></a>
         </div>
-        <div class="cv-btn">
-            <p>Get my CV</p> <img src="03_Resources/01_Graphics/02_Stock/download-btn.svg" alt="Download botton" class="download-btn">
+
+        <div class="icons-social-media-footer">
+          <a href="#"
+            ><img
+              class="icon-slackoverflow"
+              src="/03_Resources/01_Graphics/02_Stock/ic_stackoverflow.svg"
+              alt="Slack Overflow icon"
+          /></a>
+          <a href="#"
+            ><img
+              class="icon-github"
+              src="/03_Resources/01_Graphics/02_Stock/ic_github.svg"
+              alt="Github icon"
+          /></a>
+          <a href="#"
+            ><img
+              class="icon-twitter"
+              src="/03_Resources/01_Graphics/02_Stock/ic_twitter.svg"
+              alt="Twitter icon"
+          /></a>
+          <a href="#"
+            ><img
+              class="icon-linkedin"
+              src="/03_Resources/01_Graphics/02_Stock/ic_linkedin.svg"
+              alt="Linkedin icon"
+          /></a>
+          <a href="#"
+            ><img
+              class="icon-angelist"
+              src="/03_Resources/01_Graphics/02_Stock/ic_angellist.svg"
+              alt="Angelist icon"
+          /></a>
+          <a href="#"
+            ><img
+              class="icon-medium"
+              src="/03_Resources/01_Graphics/02_Stock/ic_medium.svg"
+              alt="Medium icon"
+          /></a>
```

Comment:
- [ ] Kindly make sure all the image links provided are not broken to match the Figma design😎
- Your design 👇
![screenshot (30)](https://user-images.githubusercontent.com/44624138/149046109-0caf11c1-a48d-4f78-bcf6-ff586a937c48.png)